### DISCUSSION TITLE
Adventure map, last opponent loses the game, window display sequence adjustment

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#10299] [General]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.
- [X] I've checked that this issue is reproducible on the [**latest snapshot build**](https://ihhub.github.io/fheroes2/INSTALL.html#snapshots-latest-builds).

### Platform

Windows

### Describe the bug

When a player wins the scenario because the last opponent loses the game, the window display sequence should be like this:

Player's turn window:
![Win lose game 003](https://user-images.githubusercontent.com/43583994/235463558-e35a45cd-ac28-40d9-ab33-c757d63205a4.png)

New week (or new month) window if applicable:
![Win lose game 004](https://user-images.githubusercontent.com/43583994/235463577-71a22641-8b78-4df4-a354-9663fd6a8515.png)

Window of last opponent losing the game:
![Win lose game 005](https://user-images.githubusercontent.com/43583994/235463619-65d52154-4037-460a-821c-dc30040cecfb.png)
In this case the window yellow title should be:
"Victory!"

Instead of:
"Major Event!"

Here, the blue player was indeed defeated, so it is indeed a major event, but at the same time it is also the victory of the green player.

Score screen:
![Win lose game 008](https://user-images.githubusercontent.com/43583994/235463691-7a01a646-bf3b-4905-a6da-1406b0246ede.png)

Eventually, in this case, the new week (or month) window might not be displayed because the green player wins the game anyway.
What do you think about this?

In fheroes2:

https://user-images.githubusercontent.com/43583994/235457617-586bb7d2-f9af-449d-ab25-803185dc7571.mp4

### Save file

Here are the save files:
[Last day 01.zip](https://github.com/ihhub/fheroes2/files/11365399/Last.day.01.zip)

### Additional info

Related issue: https://github.com/ihhub/fheroes2/issues/7091.

**Comment by oleg-derevenetz** [comment]
These "[color] has been vanquished" messages are in fact shown "between" days before any player gets a turn during the new day, because they are in fact addressed to all players, not just the first one. They doesn't mean any "victory" per se because all they show is that some particular player has been vanquished at the onset of the corresponding day, but there still may be another active players, so no "Victory!" title is applicable here.

**Comment by oleg-derevenetz** [comment]
> Eventually, in this case, the new week (or month) window might not be displayed because the green player wins the game anyway.
> What do you think about this?

The reason behind the "new week" duties (including display of regular resource income) are performed first and check for victory conditions is performed next is the "Accumulate Gold" victory condition.

**Comment by ihhub** [comment]
Moving this issue to the Discussion topic as it doesn't seem that we found a consensus on a solution.
